<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pikture</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <widget class="QMenu" name="menuTransformations">
     <property name="title">
      <string>Transformations</string>
     </property>
     <addaction name="actionZoom_avant"/>
     <addaction name="actionZoom_arriere"/>
     <addaction name="separator"/>
     <addaction name="actionMirroir_vertical"/>
     <addaction name="actionMiroir_horizontal"/>
     <addaction name="separator"/>
     <addaction name="actionRotation_horaire"/>
     <addaction name="actionRotation_anti_horaire"/>
    </widget>
    <addaction name="actionDiaporama"/>
    <addaction name="separator"/>
    <addaction name="menuTransformations"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actionPr_f_rences"/>
    <addaction name="actionInformations_fichier"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuOutils"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionDiaporama">
   <property name="text">
    <string>Diaporama</string>
   </property>
  </action>
  <action name="actionPr_f_rences">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
  <action name="actionInformations_fichier">
   <property name="text">
    <string>Informations fichier</string>
   </property>
  </action>
  <action name="actionZoom_avant">
   <property name="text">
    <string>Zoom avant</string>
   </property>
  </action>
  <action name="actionZoom_arriere">
   <property name="text">
    <string>Zoom arrière</string>
   </property>
  </action>
  <action name="actionMirroir_vertical">
   <property name="text">
    <string>Miroir vertical</string>
   </property>
  </action>
  <action name="actionMiroir_horizontal">
   <property name="text">
    <string>Miroir horizontal</string>
   </property>
  </action>
  <action name="actionRotation_horaire">
   <property name="text">
    <string>Rotation horaire</string>
   </property>
  </action>
  <action name="actionRotation_anti_horaire">
   <property name="text">
    <string>Rotation anti horaire</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
